src/access.ts

```diff
@@ -137,6 +137,12 @@ export class WebAccessFS extends Async(IndexFS) {
 		}
 	}
 
+	async readdir(path: string): Promise<string[]> {
```

Comment:
Thanks. It could potentially be useful for perf. to cache this, but I think for now this is good.